client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -362,37 +400,25 @@ describe('Edit Required Information', () => {
       }
     };
     const storeWithFields = mockStore({ profile: stateWithFields });
+    stateWithFields['data']['demographics']['phone'] = ('122','224','2942-292992');
```

Comment:
@williamkwon97 Is a tuple intended here? JavaScript interprets this as the first value. If you want to iterate over an array, we'll need a loop of some sort